<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-2C0613A2" name="Test2" tests="1" failures="0" timestamp="10 Jan 2022 07:07:17 GMT" time="0.058" errors="0">
  <testcase name="@BeforeTest intialize" time="0.0" classname="EndtoEndFrameWork.validateTitleTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at resources.Base.initializeDriver(Base.java:39)
	at EndtoEndFrameWork.validateTitleTest.intialize(validateTitleTest.java:25)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest intialize -->
  <testcase name="@AfterTest tearDown" time="0.0" classname="EndtoEndFrameWork.validateTitleTest">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="ValidateTitle" time="0.013" classname="EndtoEndFrameWork.validateTitleTest">
    <skipped/>
  </testcase> <!-- ValidateTitle -->
</testsuite> <!-- Test2 -->
